currency-v1/apiproxy/proxies/default.xml

```diff
@@ -2,7 +2,8 @@
 <ProxyEndpoint name="default">
 
 	<DefaultFaultRule name="all">
-		<AlwaysEnforce>true</Always
+		<AlwaysEnforce>true</AlwaysEnforce>
+		<!-- i fixed my bug -->
```

Comment:
why did you leave this comment in?

actually, its release day - just leave it.